### PR TITLE
Fix sublinks display on mobile

### PR DESCRIPTION
**Pull request by gregberge**
CleanShot 2024-12-11 at 14 20 31@2x
CleanShot 2024-12-11 at 14 20 43@2x

### CODE REVIEW

**Review by scazan — APPROVED** [review]

File: packages/gitbook/src/components/Header/Dropdown.tsx [hunk 1]
```diff
@@ -111,31 +111,38 @@ export function DropdownMenu(props: { children: React.ReactNode }) {
  * Menu item in a dropdown.
  */
 export function DropdownMenuItem(props: {
-    href: string;
+    href: string | null;
     active?: boolean;
+    className?: ClassValue;
```
Comment (clairechabas) [inline packages/gitbook/src/components/Header/Dropdown.tsx:116]: I'm not familiar with the GBO dev env yet : why isn't that a string?

File: packages/gitbook/src/components/Header/Dropdown.tsx [hunk 2]
```diff
@@ -111,31 +111,38 @@ export function DropdownMenu(props: { children: React.ReactNode }) {
...
 }) {
-    const { children, active = false, href } = props;
+    const { children, active = false, href, className } = props;
+
+    if (href) {
+        return (
+            <Link
+                href={href}
+                prefetch={false}
+                className={tcls(
```
Comment (clairechabas) [inline packages/gitbook/src/components/Header/Dropdown.tsx:126]: Curious to know why we went for `tcls` instead of `clsx` on GBO?

**Review by clairechabas — APPROVED** [review]
much better 👍

File: packages/gitbook/src/components/Header/Dropdown.tsx (hunk 2 above)
Comment (gregberge) [reply packages/gitbook/src/components/Header/Dropdown.tsx:126]: It's a mistake, but it's like that from the beginning. At some point we should change it to improve perfs.

File: packages/gitbook/src/components/Header/Dropdown.tsx (hunk 1 above)
Comment (gregberge) [reply packages/gitbook/src/components/Header/Dropdown.tsx:116]: We manipulate `clsx` values, but IMO we should not. All components should only accept a `className?: string` and if you need to merge thing you can do it when you pass the prop: `<Any className={clsx(...)} />` like we do in the app.

**Pull request merged**